service/billing/model.go

```diff
@@ -0,0 +1,532 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package billing
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// Budget configuration to be created.
+type Budget struct {
+	Alerts []BudgetAlert `tfsdk:"alerts"`
```

Comment:
I'm wondering if list can be done with `types.List` from the tfsdk?

Reply:
Looks like it has the same problem with map?

Reply:
Yep, It has same problem like map.